src/history/provider.ts

```diff
@@ -1,10 +1,11 @@
-import {Benchmark} from "../types.js";
+import {Benchmark, EnumLike} from "../types.js";
 
-export enum HistoryProviderType {
-  Local = "Local",
-  GaCache = "GaCache",
-  S3 = "S3",
-}
+export const HistoryProviderTypeEnum = {
```

Comment:
I see you just did the minimal diff here. But consider removing `Type` from the const and type

Reply:
I was afraid to change that considering that may trigger another discussion. :)